testing/py3-subunit/APKBUILD

```diff
@@ -0,0 +1,25 @@
+# Contributor: Robert Sacks <robert@sacks.email>
+# Maintainer: Robert Sacks <robert@sacks.email>
+pkgname="py3-subunit"
+_pyname="python-subunit"
+pkgver="1.3.0"
+pkgrel=0
+pkgdesc="Python implementation of subunit test streaming protocol"
+url="https://launchpad.net/subunit"
+arch="noarch"
+license="Apache-2.0 BSD-3-Clause"
+options="!check" # Test suite issues
```

Comment:
Please be more specific

Reply:
The issue comes from the package py-testtools. Lines like [this](https://github.com/testing-cabal/testtools/blob/master/testtools/content.py#L23), [this](https://github.com/testing-cabal/testtools/blob/master/testtools/tests/test_testsuite.py#L8), and [this](https://github.com/testing-cabal/testtools/blob/master/testtools/tests/test_compat.py#L6) will always fail on python3. There are already open and closed pull requests about this on the upstream project. Should I just describe this in a comment, or should I look into changing the py-testtools package to use sed to fix these issues as seen [here](https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/python-testtools#n23)?